src/postgres_init/schema.sql

```diff
@@ -0,0 +1,32 @@
+-- On creation we are connected to development
+CREATE DATABASE development;
+
+\c development;
+
+REVOKE ALL ON DATABASE development FROM public;
+
+-- The role used by the api
+CREATE ROLE read_write_role WITH LOGIN PASSWORD 'password';
+
+GRANT CONNECT ON DATABASE development TO read_write_role;
+
+GRANT USAGE ON SCHEMA public TO read_write_role;
+
+-- Gives these privileges to read_write_role on any table that we create in public
+ALTER DEFAULT PRIVILEGES 
+  FOR ROLE read_write_role
+  IN SCHEMA public
+  GRANT SELECT, INSERT, UPDATE, DELETE ON TABLES TO read_write_role;
+
+-- Tables
```

Comment:
is there a way to make it so that we use the same schema for creating tables in here as we will use for staging and production?

Reply:
This is a temporary schema to test, we are going to have the real schema (probably using knex instead of raw sql) in iac and pull it from there.

Reply:
great!